File: Heat/ViewModels/Invoices/InvoicePaymentViewModel.vb
Imports System.ComponentModel.DataAnnotations
Imports Heat.ViewModels

Namespace ViewModels.Invoices

    Public Class InvoicePaymentViewModel

        <Key> _
        Property ID As Integer

        <Display(name:="Cliente")> _
        Property CustomerName As String

        <Display(name:="Numero fattura provvisoria")> _
        Property InvoiceNumber As String

        <Display(name:="Data documento")> _
        Property InvoiceDate As String

        Property Rows As List(Of InvoiceRowViewModel)

        <Required> _
        <Display(name:="Condizioni di pagamento")> _
        Property PaymentID As Integer

        Property PaymentList As IEnumerable(Of SelectListItem)

        <Display(name:="Fattura esente")> _
        Property IsTaxExempt As Boolean

        <Display(name:="Esenzione")> _
        Property TaxExemption As String


    End Class

End Namespace

File: Heat.ConvertedToC#/ViewModels/Invoices/InvoicePaymentViewModel.cs
using System.Collections.Generic;
using System.Web.Mvc;
using System.ComponentModel.DataAnnotations;

namespace Heat.ViewModels.Invoices
{

    public class InvoicePaymentViewModel
	{

		[Key()]
		public int ID { get; set; }

		[Display(Name = "Cliente")]
		public string CustomerName { get; set; }

		[Display(Name = "Numero fattura provvisoria")]
		public string InvoiceNumber { get; set; }

		[Display(Name = "Data documento")]
		public string InvoiceDate { get; set; }

		public List<PresentationInvoiceRowViewModel> Rows { get; set; }
		//Property Rows As List(Of InvoiceRowViewModel)

		[Required()]
		[Display(Name = "Condizioni di pagamento")]
		public int PaymentID { get; set; }

		public IEnumerable<SelectListItem> PaymentList { get; set; }

		[Display(Name = "Fattura esente")]
		public bool IsTaxExempt { get; set; }

		[Display(Name = "Esenzione")]
		public string TaxExemption { get; set; }


	}

}
